entries/w/wefunder.com.json

```diff
@@ -2,9 +2,11 @@
     "Wefunder": {
         "domain": "wefunder.com",
         "tfa": [
+            "totp",
```

Comment:
Looks like SMS is also supported.
![](https://static.helpjuice.com/helpjuice_production/uploads/upload/image/3368/direct/1696978436674-image.png)
```suggestion
            "totp",
            "sms",
```